srcpkgs/glm/template

```diff
@@ -30,6 +32,6 @@ post_install() {
 		dos2unix "$f"
 	done
 	rm -f "${DESTDIR}/usr/include/CMakeLists.txt"
-	sed -n '88,110p' manual.md > LICENSE
+	mv copying.txt LICENSE
 	vlicense LICENSE
```

Comment:
use `vlicense` to install the license, the 2nd argument can be used to change the filename

Reply:
My bad, somehow missed that when I looked in the packages manual. Should be fixed now